src/lib/address/utils.ts

```diff
@@ -1 +1,10 @@
+import { isAddress } from 'viem'
+import { getEthersProvider } from 'lib/ethersProviderSingleton'
+
 export const isEthAddress = (address: string): boolean => /^0x[0-9A-Fa-f]{40}$/.test(address)
+
+export const isSmartContractAddress = async (address: string): Promise<boolean> => {
+  if (!isAddress(address)) throw new Error('Invalid address')
```

Comment:
we pass `''` into this function:
[![image](https://github.com/shapeshift/web/assets/125113430/a873c60f-5e33-4dd8-a6bb-a9ab282d9555)](https://github.com/shapeshift/web/pull/5716/files#diff-e4e72fac2893f15d931375d897595ea98dd219d21c3b04cb3ff58fc11ff0d341R767)

so we might want to prevent this throwing:
```suggestion
  if (!isAddress(address)) return false
```